File: haskell/fizzbuzz.hs
fizzbuzz :: Integer -> String
fizzbuzz i
  | (mod i 15) == 0  = "FizzBuzz"
  | (mod i 5)  == 0  = "Buzz"
  | (mod i 3)  == 0  = "Fizz"
  | otherwise        = show i

main = do mapM_ (putStrLn . fizzbuzz ) [1..100]

File: julia/fizzbuzz.jl
for i = 1:100
    if mod(i, 15) == 0
        println("FizzBuzz")
    elseif mod(i, 3) == 0
        println("Fizz")
    elseif mod(i, 5) == 0
        println("Buzz")
    else
        println(i)
    end
end
